invenio_app_rdm/records_ui/views/filters.py

```diff
@@ -190,9 +190,14 @@ def custom_fields_search(field, field_value, field_cfg=None):
         namespace_string = r"\:".join(namespace_array) + f".{localised_title}.{locale}"
     else:
         namespace_string = r"\:".join(namespace_array)
+        # Check if the field config has a landing page search attribute to search by
+        landing_page_search_attr = (field_cfg or {}).get("landing_page_search_attr", False)
+        if landing_page_search_attr:
+            namespace_string += f".{landing_page_search_attr}"
```

Comment:
this should be 
```suggestion
            namespace_string += r".".format({landing_page_search_attr})
```
no? To avoid the warning?